env.sh

```
@@ -0,0 +1,152 @@
+        --env=*)
+            ENV="${1#*=}"  # Extract value after '=' character
+            validate_env "$ENV"
+            shift
+            ;;
+        --infisical-token=*)
+            TOKEN="${1#*=}"  # Extract value after '=' character
+            shift
+            ;;
+        --)
+            shift
+            COMMAND="$@"  # Everything after -- becomes the command to execute
```

Shellcheck (0.10.0)

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)